examples/pipelines/disk-uploader/example-vm-exported-task.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1
```

Comment:
use create-vm-from-manifest task instead of defining new task

Reply:
Ah good catch. Sure thing!